.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pnpm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

It might be handy to get Dependbot updates for GH actions also

```suggestion
      interval: "weekly"

    - package-ecosystem: "github-actions"
      directory: "/"
      schedule:
        interval: "weekly"
```